### PR TITLE
Update some deps

### PR DESCRIPTION
**Pull request by ahuth**
This PR updates some deps.

### CODE REVIEW

File: .storybook/pages/AlongDemo/AlongDemo.stories.ts [hunk 1]
```diff
@@ -19,4 +19,11 @@ export const WatchPage: StoryObj<Args> = {
     const nextPageButton = await canvas.findByText('Hogwarts');
     userEvent.click(nextPageButton);
   },
+  parameters: {
+    axe: {
+      // Don't run axe on this story. The video seems to cause asset preloading to fail (even when
+      // the video has preload="none").
```
Comment (ahuth) [inline .storybook/pages/AlongDemo/AlongDemo.stories.ts:25]: Fails at https://github.com/dequelabs/axe-core/blob/d4522cdd7a90018336098f9b2119e353bd5a71ee/lib/core/utils/preload.js#L105

File: package.json [hunk 2]
```diff
@@ -127,22 +127,22 @@
     "@types/react-beautiful-dnd": "^13.1.3",
     "@types/react-dom": "^18.0.11",
     "@types/react-portal": "^4.0.4",
-    "axe-core": "4.4.3",
-    "chromatic": "^6.17.0",
+    "axe-core": "4.6.3",
```
Comment (jinlee93) [inline package.json:130]: `^4.6.3`? 
Iirc we kept it at 4.4.3 due to timeouts but it looks like we're just skipping that. Can we remove `axe-core` from the ignore in the [renovate config](https://github.com/chanzuckerberg/edu-design-system/blob/main/.github/renovate.json5#L24-L25)?

Comment (ahuth) [reply package.json:130]: 👍

**Review by jinlee93 — APPROVED** [review]
Sweet thanks!

File: package.json (hunk 2 above)
Comment (jinlee93) [reply package.json:130]: Can we add the caret back (shoulda been more clear lol sorry)

**Pull request merged**